moto/core/exceptions.py

```diff
@@ -44,6 +45,9 @@ class RESTError(HTTPException):
     # most APIs use <RequestId>, but some APIs (including EC2, S3) use <RequestID>
     request_id_tag_name = "RequestId"
 
+    # When this field is set, the `Type` field will be included in the response
+    sender_fault = True
```

Comment:
Minor nitpick - but the attribute-name doesn't make it immediately clear to me what it does. How about `include_type_sender`?

Reply:
Changed in 225023df6 :+1: